cmd/snap-bootstrap/bootstrap/bootstrap.go

```diff
@@ -309,3 +319,27 @@ func isCompatibleSchema(gadgetSchema, diskSchema string) bool {
 		return false
 	}
 }
+
+func getModel() (*asserts.Model, error) {
+	modeenv, err := boot.ReadModeenv("")
```

Comment:
reading Modeenv from here is a bit weird, I think the model should be passed in from snapd actually

Reply:
You mean the pass the label, and read model from seed locally? Or somehow pass the model itself?